src/search.c

```diff
@@ -499,6 +578,26 @@ static void do_search(struct manifest *MoM, char search_type, char *search_term)
 			continue;
 		}
 
+		/* record contentsize and includes for install size calculation */
```

Comment:
> Can you not just add subman onto a linked list

I could, but need to track whether I've `seen` that bundle and if that bundle `is_tracked` on the system. I opted to modify the local `bundle_result` struct instead of adding two more fields to the `manifest` struct.